### DISCUSSION TITLE
Code Wars JavaScript output

### DISCUSSION THREAD

**Discussion by Esraa-Saeed** [codewars/codewars.com#2872] [Q&A]
Hello all,

I'm a beginner learning about JavaScript. I've been using Code Wars educational platform to practice problems with the help of some online tutors. In the output screen I only get that I've passed or not, but I should be seeing the output too.

Anyone can help, please?
![Capture](https://user-images.githubusercontent.com/116196283/204671094-2cfe797d-85e9-4d67-850b-c995ff744ade.JPG)

**Comment by JohnMeyerhoff** [comment]
AFAIK this should be set by the developer of the kata. He or she can enable this output or disable it.

**Reply by Esraa-Saeed** [reply]
Right! I noticed this later after trying more tests. Thank you 😊

**Comment by JohanWiltink** [comment]
You do _not_ need to see the output. You can't learn from succeeding tests, you can only learn from failing tests. Failing tests ( normally ) show outputs ( and hopefully inputs, but you can always print those yourself ).

All this assumes tests correctly and completely validate your solution.

**Reply by Esraa-Saeed** [reply]
Thank you so much !
